### PR TITLE
[RFC] vim-patch.sh: List patches in chronological order.

### PR DESCRIPTION
**Pull request by fwalch**

### CODE REVIEW

**Comment by justinmk** [comment]
Personally I'm ok with the 1000+ range sorting at the top, using "C" sort behavior instead of natural range, if it is more reliable across systems.

**Comment by justinmk** [comment]
Merged the NA fix.

**Comment by fwalch** [comment]
@justinmk How about now? Too slow?

File: scripts/vim-patch.sh [hunk 1]
```diff
@@ -161,29 +161,19 @@ list_vim_patches() {
...
-    local vim_runtime_commits=$(cd "${VIM_SOURCE_DIR}" && \
-      git log --reverse --format='%H' --grep='^patch' --grep='^updated for version' \
-        --invert-grep e2719096250a19ecdd9a35d13702879f163d2a50..HEAD)
-  else # --invert-grep requires git 2.4+
-    echo "Warning: some runtime updates may not be listed (requires git 2.4+)."
-    local vim_runtime_commits=$(cd "${VIM_SOURCE_DIR}" && \
-      git log --reverse --format='%H' --grep='Updated' e2719096250a19ecdd9a35d13702879f163d2a50..HEAD)
-  fi
+  # Get commits since 7.4.602.
+  local vim_commits=$(cd "${VIM_SOURCE_DIR}" && git log --reverse --format='%H' v7.4.602..HEAD)
```
Comment (justinmk) [inline scripts/vim-patch.sh]: I think this is fine, because if it's too slow it means we're too far behind. We can bump it forward as we catch up.

**Comment by justinmk** [comment]
Merged. Thanks @fwalch

**Pull request closed without merging**